scripts/create.js

```diff
@@ -4,8 +4,8 @@ const NAME = `create.js`;
 
 console.log(`RUNNING: ${NAME}`);
 
-const createRun = DB.schema.createTable('filecoin_storage_providers', function (table) {
-  table.uuid('id').primary().unique().notNullable().defaultTo(DB.raw('uuid_generate_v4()'));
+const createFilecoinStorageProvidersTable = DB.schema.createTable('filecoin_storage_providers', function (table) {
+  table.uuid('id').primary().unique().notNullable().defaultTo(DB.raw('gen_random_uuid()'));
```

Comment:
@jimmylee what version of postgres are we using in prod? the `uuid_generate_v4()` function wasn't playing nice with the docker image as it required the `uuid` extension to be installed, but it's builtin as `gen_random_uuid()` for v13 and onwards.

Reply:
We're using 14. you can fix this by

`Promise.all([db.raw('CREATE EXTENSION IF NOT EXISTS "uuid-ossp"')]);`

And I would just add a setup script that looks like

```
import configs from "../knexfile";
import knex from "knex";

const name = `database-setup.js`;
const environment =
  process.env.NODE_ENV !== "production" ? "development" : "production";
const envConfig = configs[environment];
const db = knex(envConfig);

console.log(`RUNNING: ${name} NODE_ENV=${environment}`);

Promise.all([db.raw('CREATE EXTENSION IF NOT EXISTS "uuid-ossp"')]);

console.log(`FINISHED: ${name} NODE_ENV=${environment} (⌘ + C to quit)`);
```

Into the codebase